CHANGELOG.md

```diff
@@ -2,9 +2,18 @@
 
 ## master
 
+- [#32](https://github.com/SuperGoodSoft/solidus_taxjar/pull/32) Add Taxjar customers and exempt regions integration
 - [#21](https://github.com/SuperGoodSoft/solidus_taxjar/pull/21) Migrated project to `solidus_dev_support`
 - [#22](https://github.com/SuperGoodSoft/solidus_taxjar/pull/22) Added support for TaxJar address validation API through `SuperGood::SolidusTaxJar::Addresses` class
 
+**Breaking Changes**:
```

Comment:
What do you think of moving the "upgrade instructions" to here instead of the readme? If there's future breaking changes, the readme could get very full of upgrade instructions very fast. This seems like it could be a good place for them instead!

Reply:
I can move it if that's what you all want. Doesn't bother me. I put it where it made sense at the time.